spec/poker_hands_spec.rb

```diff
@@ -2,3 +2,4 @@
   it "has a version number" do
     expect(PokerHands::VERSION).not_to be nil
   end
+end
```

Comment:
Thanks for the fix, but this is a tiny bit out of place in this commit. Would make sense to break this out next time.